tasks/Win32NT/tarball_install.yml

```diff
@@ -0,0 +1,101 @@
+---
+
+- name: Check zookeeper version
+  fail:
+    msg: "Unsupported version for this platform!"
+  when: zk_version is version_compare('3.5.5', '==')
+
+- name: Check if '{{ zk_dir }}' folder exists
+  win_stat:
+    path: '{{ zk_dir }}'
+  register: zk_folder
+
+- name: Install zookeeper from tarball
+  block:
+
+  - name: Install PSCX
+    win_psmodule:
+      name: Pscx
+      state: present
+      allow_clobber: true
+
+  - name: 'Get artifact checksum file'
+    win_uri:
+      url: '{{ zk_url }}.{{ zk_checksum }}'
+      return_content: true
+    register: artifact_checksum_file
+
+  - name: Debug artifact_checksum_file
+    debug:
+      var: artifact_checksum_file
+
+  - name: Get artifact checksum
+    set_fact:
+      artifact_checksum: '{{ artifact_checksum_file.content.split(" ")[0] }}'
```

Comment:
```suggestion
      artifact_checksum: '{{ artifact_checksum_file.content | trim }}'
```

Reply:
Iam not trying to remove trailing whitespaces, I need to get a first part of a string:
b14f7a0fece8bd34c7fffa46039e563ac5367607c612517aa7bd37306afbd1cd  zookeeper-3.4.14.tar.gz